pkg/rule/criteria.go

```diff
@@ -27,20 +27,45 @@ type Criteria struct {
 
 // CriteriaUser holds all relevant matchers concerning a user.
 type CriteriaUser struct {
-	Age *matcher
-	ID  *MatcherStringList
+	Age          *matcher
```

Comment:
Shouldn't this be a `MatcherInt` as well?

Reply:
you're right. we're not using it though but I'll change it